docs/static_landing_page/assets/css/style.css

```diff
@@ -0,0 +1,932 @@
+        line-height: 36px;
+    }
+    #hero h2 {
+        font-size: 18px;
+        line-height: 24px;
+        margin-bottom: 30px;
+    }
+    #hero p {
+        padding: 10px 0px 0px 0px;
+    }
+    #hero .hero-img img {
+        width: 70%;
+    }
+}
+
+@media (max-width: 575px) {
+    #hero {
+        text-align: center;
+    }
+    #hero ul {
+        text-align: left;
+        font-size: 14px;
+    }
+    #hero .hero-img img {
+        width: 80%;
+    }
+    #hero .btn-get-started,
+    #hero .btn-get-quote {
+        padding-left: 18px;
+        padding-right: 18px;
+        font-size: 14px;
+    }
+}
+
+@media (max-width: 1024px) {
+    #hero h1 {
+        padding: 0px;
+    }
+}
+
+
+/*--------------------------------------------------------------
+# Sections General
+--------------------------------------------------------------*/
+
+.section-bg {
+    background-color: #eeeef5;
+}
+
+.section-title h1 {
+    font-family: 'TT Commons-regular', sans-serif;
+    font-size: 48px;
+    line-height: 52px;
+    letter-spacing: 0.015em;
+    color: #333333;
+}
+
+section p {
+    font-family: 'TT Commons-regular', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 26px;
+}
+
+.section-title h2 {
+    font-size: 32px;
+    font-weight: bold;
+    text-transform: uppercase;
+    margin-bottom: 20px;
+    padding-bottom: 20px;
+    position: relative;
+    color: #434175;
+}
+
+.section-title h2::after {
+    content: '';
+    position: absolute;
+    display: block;
+    width: 50px;
+    height: 3px;
+    background: #5a5af3;
+    bottom: 0;
+    left: calc(50% - 25px);
+}
+
+
+/*--------------------------------------------------------------
+# About
+--------------------------------------------------------------*/
+
+.about {
+    padding: 95px 0px 101px 0px;
+}
+
+.about-img {
+    /* width: 746px; */
+    height: 600px;
+    background: #F2F2F2;
+}
+
+#about .section-title {
+    margin-bottom: 100px;
+}
+
+
+/*--------------------------------------------------------------
+# Usecase
+--------------------------------------------------------------*/
+
+.usecase {
+    padding: 83px 0px;
+    background: #F2F2F2;
+    /* opacity: 0.5; */
+}
+
+.usecase-title>* {
+    text-align: center;
+}
+
+.usecase .section-title p {
+    font-size: 15px;
+    font-weight: bold;
+    letter-spacing: 0.06em;
+}
+
+.usecase-img {
+    margin-top: 50px;
+    float: right;
+    width: 430px;
+    height: 400px;
+    background: #757575;
+    opacity: 0.1;
+}
+
+.usecase-content {
+    margin-top: 50px;
+}
+
+.usecase-content button {
+    padding: 13px 30px;
+    background: #757575;
+    /* DP 10 */
+    box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.08);
+    border-radius: 30px;
+    color: #ffffff;
+}
+
+@media (max-width: 992px) {
+    .usecase-img {
+        float: none;
+        width: 100%
+    }
+}
+
+
+/*--------------------------------------------------------------
+# Getstart
```

Comment:
Is this meant to be for "getting started"?

Reply:
Yes

Reply:
Can you update the text to say getting started, please?